### DISCUSSION TITLE
CSS Battle #94 – Ripples

### DISCUSSION THREAD

**Discussion by Narigo** [Virtual-Coffee/virtualcoffee.io#1284] [CSS Battles]
### Link to battle:

[Let's battle! ⚔️](https://cssbattle.dev/play/94)

### Copy the code block below to format your comment on the discussion thread:

````md
<details>
<summary>Code Source – <strong>score {characters}</strong></summary>
<!-- have to be followed by an empty line! -->

```html
your cssbattles code here!
```

</details>
````

### What others will see:

This will result in a nice hidden bit like so:

<details>
<summary>Code Source – <strong>score {characters}</strong></summary>

```html
your cssbattles code here!
```
</details>

**Comment by Narigo** [comment]
<details>
<summary>Code Source – <strong> 677.64 {172}</strong></summary>


```html
<style>&{background:radial-gradient(1q,#F3AC3C 21q,#0000 0 30q,#F3AC3C 0 42q,#0000 0 53q,#998235 0 63q,#0000 0 76q,#F3AC3C 0 85q,#0000 0 99q,#F3AC3C 0 106q,#0000 0),#0E2E2C
```

</details>

**Comment by meg-gutshall** [comment]
<details>
<summary>First attempt – <strong>659.46 {200}</strong></summary>


```html
<style>*>*{background:radial-gradient(circle at center,#F3AC3C 0 20px,#0E2E2C 0 28px,#F3AC3C 0 40px,#0E2E2C 0 50px,#998235 0 60px,#0E2E2C 0 72px,#F3AC3C 0 80px,#0E2E2C 0 94px,#F3AC3C 0 100px,#0E2E2C 0
```

</details>

<details>
<summary>Second attempt – <strong>675.45 {175}</strong></summary>


```html
<style>*{background:radial-gradient(1q,#F3AC3C 0 5vw,#0E2E2C 0 7vw,#F3AC3C 0 5ch,#0E2E2C 0 53q,#998235 0 60px,#0E2E2C 0 9ch,#F3AC3C 0 5pc,#0E2E2C 0 94px,#F3AC3C 0 25vw,#0E2E2C
```

</details>

<details>
<summary>Third attempt (with tips from @Narigo) – <strong>678.39 {171}</strong></summary>


```html
<style>*{background:radial-gradient(1q,#F3AC3C 5vw,#0000 0 7vw,#F3AC3C 0 5ch,#0000 0 53q,#998235 0 60px,#0000 0 9ch,#F3AC3C 0 5pc,#0000 0 94px,#F3AC3C 0 25vw,#0000)#0E2E2C
```

</details>
